includes/admin/downloads/metabox.php

```diff
@@ -806,20 +810,27 @@ function edd_render_file_row( $key, $args, $post_id, $index ) {
 		'thumbnail_size' => null,
 	) );
 
-	$prices           = edd_get_variable_prices( $post_id );
-	$variable_pricing = edd_has_variable_prices( $post_id );
-	$variable_display = $variable_pricing ? '' : ' style="display:none;"';
-	$variable_class   = $variable_pricing ? ' has-variable-pricing' : ''; ?>
+	$prices           		= edd_get_variable_prices( $post_id );
+	$variable_pricing 		= edd_has_variable_prices( $post_id );
+	$variable_display 		= $variable_pricing ? '' : ' style="display:none;"';
+	$variable_class   		= $variable_pricing ? ' has-variable-pricing' : '';
+	$custom_file_options 	= isset( $wp_filter['edd_download_file_option_row'] ) ? true : false; ?>
 
 	<div class="edd-repeatable-row-header edd-draghandle-anchor">
 		<span class="edd-repeatable-row-title" title="<?php _e( 'Click and drag to re-order files', 'easy-digital-downloads' ); ?>">
 			<?php printf( esc_html__( '%1$s file ID: %2$s', 'easy-digital-downloads' ), esc_html( edd_get_label_singular() ), '<span class="edd_file_id">' . esc_html( $key ) . '</span>' ); ?>
 			<input type="hidden" name="edd_download_files[<?php echo esc_attr( $key ); ?>][index]" class="edd_repeatable_index" value="<?php echo esc_attr( $index ); ?>"/>
 		</span>
+		<?php
+		$actions = array();
+		if ( $custom_file_options ) {
+			$actions['show_advanced'] = '<a href="#" class="toggle-custom-price-option-section">' . __( 'Show advanced settings', 'easy-digital-downloads' ) . '</a>';
```

Comment:
```suggestion
			$actions['show_advanced'] = '<a class="toggle-custom-price-option-section">' . __( 'Show advanced settings', 'easy-digital-downloads' ) . '</a>';
```
This really should be a button instead of a link but I know this is following the pattern set by the existing "button"...but the `href` should be removed, pending fixing all of the buttons in the metaboxes.